content/release-notes/2023-12-12-console.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Reset branch from parent and other improvements
+---
+
+### Branch reset
+
+Neon is pleased to announce the latest branching feature to help improve your development workflows: **branch reset**.
+
+With one click, you can now update your working branch to the latest available schema and data from its parent branch, avoiding labor-intensive and error-prone manual updates or branch restorations. This action is availble from both the Neon Console and the Neon CLI. For details, see [Reset branch from parent](/docs/manage/branches#reset-a-branch-from-parent).
+
+![Reset branch from parent](/docs/relnotes/reset_from_parent.gif)
+
+### Set context for the Neon CLI
+
+Using a new `neonctl` command, `set-context`, you can now set a background context for your CLI sessions, letting you perform project or branch-specific actions without having to specify the project or branch id in every command. See [Neon CLI commands — set-context](docs/reference/cli-set-context) for more detail.
+
+### Support for timescaledb 2.13.0 for Postgres 16
+
+The `timescaledb` extension for scheduling stored procedures was updated to a newer version that includes support for Postgres 16.
+
+| Postgres extension           | Old version   | New version   |
+|------------------------------|---------------|---------------|
+| `timescaledb`                 |2.10.1        | 2.13.0        |
+
+Older versions remain on Postgres 15 or 14.
+
+If you installed this extension previously and want to upgrade to the latest version, please refer to [Update an extension version](/docs/extensions/pg-extensions#update-an-extension-version) for instructions.
+
+For a complete list of Postgres extensions supported by Neon, see [Postgres extensions](/docs/extensions/pg-extensions).
+
+### Fixes & improvements
+* UI: Fixed an unresponsive Read-only (RO) radio button option that sometimes appeared when creating new endpoints.
```

Comment:
```suggestion
* UI: Fixed an unresponsive Read-only (RO) radio button option that appears on the **Create Compute Endpoint** modal.
```

Reply:
One more related item:
This docs file needs to be updated. There's a dash (-) for PG 16. This needs to be changed to 2.13.0
https://neon.tech/docs/extensions/pg-extensions